pkg/providers/tinkerbell/tinkerbell.go

```diff
@@ -194,7 +194,8 @@ func (p *Provider) EnvMap(spec *cluster.Spec) (map[string]string, error) {
 		//
 		// Env read having set TINKERBELL_IP in the deployment manifest.
 		// https://github.com/tinkerbell/cluster-api-provider-tinkerbell/blob/main/controllers/machine.go#L192
-		"TINKERBELL_IP": "<set in eks-a tinkerbell provider>",
+		"KUBEADM_BOOTSTRAP_TOKEN_TTL": "120m",
+		"TINKERBELL_IP":               "IGNORED",
```

Comment:
Can we maintain the note about where this is set so we know where to look if its spotted in the environment.

Reply:
🤦🏻 _TINKERBELL_\_IP ... Ignore me!